packages/shared/src/utils/validation/declarative-validation-rules.js

```diff
@@ -14,7 +14,7 @@ export const validAddress = value => !/[`~!$%^&*_=+[}{\]\\"?><|]+/.test(value);
 export const validPostCode = value => value === '' || /^[A-Za-z0-9][A-Za-z0-9\s-]*$/.test(value);
 export const validTaxID = value => /(?!^$|\s+)[A-Za-z0-9./\s-]$/.test(value);
 export const validPhone = value => /^\+?((-|\s)*[0-9])*$/.test(value);
-export const validLetterSymbol = value => !/[`~!@#$%^&*)(_=+[}{\]\\/";:?><,|\d]+/.test(value);
+export const validLetterSymbol = value => /^[A-za-z]+([a-zA-Z'-.]+\s)*[a-zA-Z'-.]+$/.test(value);
```

Comment:
```suggestion
export const validLetterSymbol = value => !/[`~!@#$%^&*\s{2,})(_=+[}{\]\\/";:?><,|\d]+/.test(value);
```
it means spaces more than 2 are not accepted

Reply:
Yeap cause the backend cannot accept name ending with spaces or multiple spaces between 2 words. previously there were no error message below the textfield when there are multiple spaces between 2 words.

Reply:
I see, so it's fine.

Reply:
Orite thanks for reviewing @mahdiyeh-fs 🙏 🙇